### DISCUSSION TITLE
monero-wallet: Apple Silicon avail

### DISCUSSION THREAD

**Discussion by k0ste** [Homebrew/discussions#4789] [Casks] [answered]
Since Monero release 0.18.3.0 here https://github.com/monero-project/monero/pull/8988
Darwin aarch64 build was present by developers. Please update cask to support this 🙏

Thanks!

**Comment by SMillerDev** [comment]
Yes, please do: https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by chenrui333** [comment]
Also, the 0.18.3.0 is not being released yet.

**Reply by k0ste** [reply]
@chenrui333 https://github.com/monero-project/monero/releases/tag/v0.18.3.1
Now released 🙂

**Comment by miccal** [comment, marked as answer]
https://github.com/Homebrew/homebrew-cask/pull/157457
